thoth/user_api/api_v1.py

```diff
@@ -1062,6 +1069,7 @@ def post_build(
             "buildlog_document_id": buildlog_document_id,
         },
         202,
+        {"x-thoth-cached-document-ttl": cache_ttl},
```

Comment:
This is missing from the schema
https://github.com/thoth-station/user-api/blob/1aab302c5cb31064316bb43e7ce8fb5d893d31de/openapi/openapi.yaml#L889